jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -438,7 +439,7 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                 }
             }
 
-            add(StartBackend(gatewayHandle, remoteScriptPath, remoteProjectName, executor, lifetime, envId, isSmallInstance))
+            add(StartBackendV2(lifetime, indicator, envId, remoteProjectName))
```

Comment:
maybe we can rewrite some of the workflow to shave a couple more seconds off

Reply:
Planning to address it in another PR